### PR TITLE
Fix error when invoking CLI without command

### PR DESCRIPTION
**Pull request by maxjacobson**

### CODE REVIEW

**Comment by maxjacobson** [comment]
@pbrisbin updated - does that fix look OK? (context in commit message)

**Comment by dblandin** [comment]
@maxjacobson What do you think of resetting the resolver stubs after each spec?

```ruby
RSpec.configure do |config|
  config.after do
    ::Resolv::DefaultResolver.replace_resolvers([])
  end
end
```

File: spec/support/resolv_helpers.rb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 module ResolvHelpers
   def stub_resolv(name, address)
-    dns = double
+    dns = double("dns")
```
Comment (maxjacobson) [inline spec/support/resolv_helpers.rb]: oo sure. happy to update. This change was helpful for confirming that this was the double that was leaking, btw (just mentioning in case it wasn't clear why this change was included)
